backend/app/controllers/MenteesController.php

```diff
@@ -38,6 +42,101 @@ public function singleMentee($id) {
         ]);
     }
 
+    public function login()
+    {
+
+        $credentials = request()->get(['email', 'password']);
+
+        if(!$credentials['email'] || !$credentials['password']) {
+            response()->exit([
+                'error' => '404',
```

Comment:
this is 400, not 404. Beside I think that error status should be a number, not a string